src/xmagics/execution.cpp

```diff
@@ -123,7 +123,7 @@ namespace xcpp
             {
                 for (std::size_t n = 0; n < 10; ++n)
                 {
-                    number = std::pow(10, n);
+                    number = static_cast<size_t>(std::pow(10, n));
```

Comment:
Here I found one warning. Now the build succeeds without any Visual Studio compiler warning on machine.

Reply:
:+1: